### DISCUSSION TITLE
Venus Online Meetups / Venus线上Meetup活动

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/venus#4810] [Ideas]
## 概要

Venus线上meetup活动旨在和社区建立良好沟通和信息同步渠道。2021年12月开始**每月**都会举行一次。形式为Zoom会议，在Venus同步完信息之后，会有现场QA环节。

## 往期回顾

- 第15期[视频](https://www.youtube.com/watch?v=SKUqRNnu1yg)；
- 第14期[视频](https://www.youtube.com/watch?v=N5nI2TVmYZ0)；
- 第13期[视频](https://www.youtube.com/watch?v=2Qi3_4Q4-W8)；
- 第12期[视频](https://youtu.be/m9M7EK3_SDc)；主题：`venus`近期`sprint`新功能，新动态介绍
- 第11期[视频](https://www.youtube.com/watch?v=CALwHZePdhg)；主题：`venus`近期`sprint`新功能，新动态介绍
- 第10期[视频](https://www.youtube.com/watch?v=gXsFvqLYWz0)；主题：`fil-singapore`线下活动
- 第9期[视频](https://www.youtube.com/watch?v=4p1JvelyAvs)；主题：`venus`新`feature`简介
- 第8期[视频](https://www.youtube.com/watch?v=MkpJwmhANh4)；主题：`venus` `sprint`管理简介
- 第7期[视频](https://youtu.be/w57qEF776V4)；主题：`venus`近况
- 第六期[视频](https://www.youtube.com/watch?v=kkPrGRCwrqI&list=PLvmxExuDQi1nxFrwZGtnuURKoCNbovCpw&index=1)；主题：Venus热点接入问题QA
- 第五期[视频](https://www.youtube.com/playlist?list=PLvmxExuDQi1nxFrwZGtnuURKoCNbovCpw)；主题：Venus Master
- 第四期[视频](https://youtu.be/Tin9Y0Bk_AE?t=1506)；主题：venus-cluster详解
- 第三期[视频](https://www.youtube.com/watch?v=th8QJGFcNH8)；主题：venus-cluster初步分享
- 第二期[视频](https://www.youtube.com/watch?v=M1kaNy3lqh0)；主题：venus-market v2
- 第一期[视频](https://www.youtube.com/watch?v=jikmIzbfmTg)；主题：Venus介绍

**Comment by Imccccc** [comment]
发现录制的时候，还是经常会有卡顿，回顾看起来有点费劲

**Reply by Fatman13** [reply]
Zoom好像确实有这个问题。😂 

CC @Joss-Hua 咋整？

**Reply by Joss-Hua** [reply]
优先使用本地视频录制，清晰度及流畅度高于云录制。
